sessions/class-01/exercise/start.js

```diff
@@ -67,7 +98,16 @@ export function hasValue(item) {}
  * @param {string} name Name of the person.
  * @returns {string[]} Array where firstName is in 0th position and last name is in 1st position.
  */
-export function getFirstAndLastNames(name) {}
+export function getFirstAndLastNames(name) {
+	const testName = name.trim();
+	const fullName = testName.split(' ');
+	const fName = fullName[0];
+	fullName.splice(0, 1);
+	const lName = fullName.join(' ');
+	return [fName, lName];
+}
+
+console.log(getFirstAndLastNames('   smit   V Modi'));
 
 /**
  * Get a property from an object. If the property doesn't exist then get the
```

Comment:
This is actually tricky. See this

https://github.com/swashata/multidots-frontend-dev-course-plan/pull/1#discussion_r251199647